### PR TITLE
chore(ci): use algolia-bot for gen comments

### PR DESCRIPTION
**Pull request by millotp**
## 🧭 What and Why

We already use algolia-bot everywhere else in this repo, IMO it makes sense to also use it here

### CODE REVIEW

**Review by shortcuts — DISMISSED** [review]
good point of using the built-in one is that there's no token to manager nor permission to be careful with, but up to you

File: .github/workflows/check.yml [hunk 1]
```diff
@@ -29,6 +29,7 @@ jobs:
 
       - uses: marocchino/sticky-pull-request-comment@v2
         with:
+          GIT_TOKEN: ${{ secrets.ALGOLIA_BOT_TOKEN }}
```
Comment (shortcuts) [inline .github/workflows/check.yml:32]:
```suggestion
          GITHUB_TOKEN: ${{ secrets.ALGOLIA_BOT_TOKEN }}
```

Comment (millotp) [reply .github/workflows/check.yml]: oops

**Review by Fluf22 — APPROVED** [review]
Why not, it's a valid point, but @shortcuts has a point too

**Pull request merged**